packages/at_events_flutter/pubspec.yaml

```diff
@@ -2,9 +2,9 @@ name: at_events_flutter
 description: A Flutter plugin project to manage events (time, place and attendees) using the atPlatform.
 version: 3.1.9
 homepage: https://docs.atsign.com/
-repository: https://github.com/atsign-foundation/at_widgets/tree/trunk/at_events_flutter
+repository: https://github.com/atsign-foundation/at_widgets/tree/trunk/packages/at_events_flutter
 issue_tracker: https://github.com/atsign-foundation/at_widgets/issues
-documentation: https://atsign.dev/docs/functional_architecture/libraries/#at_events_flutter
+documentation: https://docs.atsign.com/
 
 environment:
   sdk: '>=2.12.0 <3.0.0'
```

Comment:
It should support dart 3

Reply:
@sachins-geekyants I have migrated to Dart 3